cmsimple/functions.php

```diff
@@ -2369,6 +2369,25 @@ function XH_registeredEditmenuPlugins()
     return XH_registerPluginType('editmenu');
 }
 
+/**
+ * Returns the names of the registered languagemenu plugins.
+ *
+ * @return string
+ *
+ * @since 1.7.6
```

Comment:
I'd prefer

```suggestion
 * @since 1.8.0
```

(seems GH UI is currently slightly broken, but you see what I mean)